src/main/java/com/volmit/iris/util/decree/DecreeExecutor.java

```diff
@@ -21,14 +21,45 @@
 import com.volmit.iris.core.tools.IrisToolbelt;
 import com.volmit.iris.engine.framework.Engine;
 import com.volmit.iris.engine.platform.PlatformChunkGenerator;
+import com.volmit.iris.util.format.C;
 import com.volmit.iris.util.plugin.VolmitSender;
+import org.bukkit.World;
+import org.bukkit.entity.Player;
 
 public interface DecreeExecutor {
     default VolmitSender sender()
     {
         return DecreeContext.get();
     }
 
+    default Player player()
+    {
+        if (sender().isPlayer()) {
+            return sender().player();
+        }
+        return null;
+    }
+
+    default void message(String string)
+    {
+        sender().sendMessage(string);
```

Comment:
Dont do this, its not hard to type sender().message() in commands. Its harder to maintain say if we supported multiple parameters, your just adding technical debt here. Add this stuff to the sender, not the executor. Remove all message sending methods from this decree executor and add it to the sender itself

Reply:
Does the same go for 
`#access`, `#world` and `#engine`?
Eight